components/form/CheckboxGroup.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { useFormContext } from 'react-hook-form';
-import PropTypes from 'prop-types';
-import { Label, ErrorLabel } from '.';
+import propTypes from 'prop-types';
```

Comment:
I understand the reason why you changed this, but the correct way to use this is capital `P`. See https://reactjs.org/docs/typechecking-with-proptypes.html.

```
import PropTypes from 'prop-types';
```

Anytime you are in doubt, you can check the package documentation or search online for the right thing. Please fix for other instances too 🙏🏽